cypress/e2e/ui/Alert-add-fail.cy.js

```diff
@@ -0,0 +1,33 @@
+import '@testing-library/cypress/add-commands';
+
+import { getUKFormattedDate } from '../../../src/utils/helpers';
+
+describe('ALERT ADD EXPENSE - FAIL', () => {
+  before(() => {
+    cy.intercept('GET', '**/items*', []).as('getExpenses');
+    cy.visit('/');
+    cy.wait('@getExpenses');
+  });
+
+  beforeEach(() => {
+    cy.intercept('POST', '**/items*', {
+      statusCode: 500,
+    }).as('postExpenseFail');
```

Comment:
i still find this wrapping around (basically everywhere), very hard to read. 
```javascript
cy.findByRole('alert').should('contain', 'The expense could not be added. Please try again.');

// vs

cy.findByRole('alert').should(
  'contain',
  'The expense could not be added. Please try again.'
);


// --- or ---


cy.intercept('POST', '**/items*', { statusCode: 500 }).as('postExpenseFail');

// vs

cy.intercept('POST', '**/items*', {
  statusCode: 500,
}).as('postExpenseFail');
```

i think a single line reads like a book, like a sentence. in these multiline statments my eyes jump up and down. maybe i am just old.

Reply:
I totally agree. I think though that it makes sense to have a separate PR to introduce the new `printWindth: 1000`.
I tried it while coding in this PR and it really makes reading the tests much easier. Looking forward to it!

Reply:
cool. do you have a suggestion who shall do the PR?

Reply:
@MatthewBerglund @motine: I can do this after Matt is done with the edit expenses okay?

Reply:
sure, the styling issue is not super urgent, @MatthewBerglund, @GiulianoMontisEvernest. i think it might be a good task before we do the firebase switch, becauase that feature will require touching a few files.